src/Microsoft.AspNet.Mvc.Razor/ITagHelperActivator.cs

```diff
@@ -13,8 +13,9 @@ public interface ITagHelperActivator
         /// <summary>
         /// When implemented in a type, activates an instantiated <see cref="ITagHelper"/>.
         /// </summary>
-        /// <param name="tagHelper">The <see cref="ITagHelper"/> to activate.</param>
+        /// <typeparam name="TTagHelper">The <see cref="ITagHelper"/> type.</typeparam>
+        /// <param name="tagHelper">The <see cref="TTagHelper"/> to activate.</param>
         /// <param name="context">The <see cref="ViewContext"/> for the executing view.</param>
-        void Activate(ITagHelper tagHelper, ViewContext context);
+        void Activate<TTagHelper>(TTagHelper tagHelper, ViewContext context) where TTagHelper : ITagHelper;
```

Comment:
nit: existing issue but should have `[NotNull]` on both parameters, matching the implementation